src/bin/README.md

```diff
@@ -0,0 +1,13 @@
+# benchwrapper
```

Comment:
Thinking out loud - should we just move this to a `tools` directory and write it in JavaScript?

Reply:
Sure - I don't mind either way. Would y'all discuss amongst yourself and let me know what's best? (I don't have a stake in the game)

Reply:
I would prefer to create the tools dir with pure JavaScript, but defer to @bcoe for the final  call

Reply:
Chatted with @bcoe - let's create a `bin` folder in the root of the repo, and put the `*.js` and `README` file inside of there.  We can just use JavaScript.

Reply:
add to that, each dependency should be saved with `--save-dev`, so that they're in the `devDependencies` stanza rather than the `dependencies` stanza.

I still think it would be good too add a script tag:

```
"script": {
  "test": "...",
   "benchmark": "./bin/benchmark.js" 
}
```

The default exit of `benchmark.js` could then print for `.usage`:

```
npm run benchmark -- --port=[port to connect to]
```